docs/section5/Rule5-1.md

```diff
@@ -20,44 +20,42 @@
+      ```elsif ( surface_azimuth_b > 45 ) AND ( surface_azimuth_b <= 135 ): total_fenestration_area_east += fenestration_area_b```  
 
-      ```elsif ( surface_azimuth > 135 ) AND ( surface_azimuth <= 225 ): total_fenestration_area_south += fenestration_area_baseline```  
+      ```elsif ( surface_azimuth_b > 135 ) AND ( surface_azimuth_b <= 225 ): total_fenestration_area_south += fenestration_area_b```  
 
-      ```else total_fenestration_area_west += fenestration_area_baseline```  
+      ```else total_fenestration_area_west += fenestration_area_b```  
 
 - Check if the total vertical fenestration areas for the four orientation differs by less than 5%: ```if max(total_fenestration_area_north, total_fenestration_area_east, total_fenestration_area_south, total_fenestration_area_west) >= 1.05 * min(total_fenestration_area_north, total_fenestration_area_east, total_fenestration_area_south, total_fenestration_area_west)): is_rotation_required = TRUE```
 
-**Rule Assertion:** Baseline RMR Building:is_rotated = expected value: ```is_rotated_baseline == is_rotation_required```
+**Rule Assertion:** Baseline RMR Building:is_rotated = expected value: ```is_rotated_b == is_rotation_required```
 
-- Case 1, the total vertical fenestration area per exposure differ by less than 5% and the baseline building is not rotated: ```( is_rotation_required == FALSE ) AND ( is_rotated_baseline == FALSE ): PASS```  
+- Case 1, the total vertical fenestration area per exposure differ by less than 5% and the baseline building is not rotated: ```( is_rotation_required == FALSE ) AND ( is_rotated_b == FALSE ): PASS```  
 
-- Case 2, the total vertical fenestration area per exposure differ by less than 5% and the baseline building is rotated: ```( is_rotation_required == FALSE ) AND ( is_rotated_baseline == TRUE ): FAIL```  
+- Case 2, the total vertical fenestration area per exposure differ by less than 5% and the baseline building is rotated: ```( is_rotation_required == FALSE ) AND ( is_rotated_b == TRUE ): FAIL```  
 
-- Case 3, the total vertical fenestration area per exposure differ by 5% or more and the baseline building is not rotated: ```( is_rotation_required == TRUE ) AND ( is_rotated_baseline == FALSE ): MANUAL_CHECK```  
+- Case 3, the total vertical fenestration area per exposure differ by 5% or more and the baseline building is not rotated: ```( is_rotation_required == TRUE ) AND ( is_rotated_b == FALSE ): MANUAL_CHECK```  
 
-- Case 4, the total vertical fenestration area per exposure differ by 5% or more and the baseline building is rotated: ```( is_rotation_required == TRUE ) AND ( is_rotated_baseline == TRUE ): PASS```  
+- Case 4, the total vertical fenestration area per exposure differ by 5% or more and the baseline building is rotated: ```( is_rotation_required == TRUE ) AND ( is_rotated_b == TRUE ): PASS```  
```

Comment:
•	Uses is_rotated schema element that has been eliminated, should replace with ASHRAE229: building_rotation_angles
•	Rule assertion does not align with the RDS logic, which focuses on whether the building was rotated and does not confirm that results were averaged.  
•	Surfaces are now attributes of zones, not spaces, need to change accordingly
•	RDS currently considers fenestration on any walls adjacent to exterior. Instead, it needs to consider fenestration on exterior and semi-exterior walls only.  Need to update to reference Function 3: Surface Conditioning Category. See definitions below for reference. 
fenestration: an assembly, including the frame, in the building envelope that allows light to pass.  
building envelope: the exterior plus the semiexterior portions of a building. For the purposes of determining building envelope requirements, the classifications are defined as follows:
exterior building envelope: the elements of a building that separate conditioned spaces from the exterior.
semiexterior building envelope: the elements of a building that separate  conditioned space from unconditioned space or that enclose semiheated spaces through which thermal energy may be transferred to or from the exterior, to or from unconditioned spaces, or to or from conditioned spaces.
•	RDS should aggregate fenestration with azimuths that exactly match, and then determine whether max and min across all azimuths differ by more than 5%. Currently, it aggregates fenestration area that has azimuths within 90 degrees before running the 5% check.

Reply:
RDS is updated but not completed for the following: @JasonGlazer @mkarpman @supriyagoel 
1. Not sure how we can check if the baseline performance is averaged. Does OverallSimulationOutputs contain separate result set for each ASHRAE229.building_rotation_angles plus the average result?
2. Right now the code is updated to check exact azimuth. Sometimes the wall azimuth can be slightly different in the model (e.g. 0.5deg, etc) depending on the software snapping function even though they are the same on arch drawing, which could easily cause difference >5%. Not sure if this will be a concern.

Reply:
@xingcx , to your first question - I suggested some updates to schema to support the baseline energy use calculations that would be necessary for this RDS. Perhaps you should hold off with it till it schema WG chimes in. 

For the second question, we discussed early on using rounding to allow values to not be exactly the same. @supriyagoel , do we have decision on this? Seems like number of decimal digits that we want to keep depends on the attribute - e.g. it may be different for U-factors vs LPDs vs azimuths.

Reply:
@mkarpman and @xingcx regarding #1, yes- we might need to wait for the schema WG to discuss that. we could address all other comments on this RDS and then come back to it once this aspect is discussed.
Regarding #2, lets discuss this during the team call tomorrow. For the number of decimal digits, the RCT team discussed this and thought it would be best to follow 90.1 guidance on this, i.e. if 90.1 uses 3 digits, we would round to that etc. However, there is also the question of tolerance. Do we need number to match exactly after rounding off, or is a certain % variation acceptable?

Reply:
Regarding (1), please describe what is needed in OverallSimulationOutput as a new issues here: 

https://github.com/open229/ruleset-model-report-schema/issues

Reply:
@JasonGlazer I added new issues to the schema and created a shared googedoc with the proposed changes. @xingcx let's wait with this portion of the RDS till proposed changes to the schema are vetted by the WG. You are shared on the googedoc with the proposed schema, so please disregard the Schema Data Element tab in the Envelope TCD v8 spreadsheet.

@supriyagoel , concerning #2, I agree that where there is a corresponding value in 90.1, we should stick with the same number of decimal digits. But in some cases it will not be the case - e.g. with azimuths, envelope dimentions or temperatures.